spec/solver_spec.rb

```diff
@@ -9,37 +9,37 @@
     end
 
     describe "#reverse" do # test for reverse method
-    it "returns olleh for hello" do
-        string_mock = double("string_mock")
-        expect(string_mock).to receive(:reverse).and_return("olleh")
-        
-        allow(Solver).to receive(:new).and_return(string_mock)
-        
-        solver = Solver.new
-        result = solver.reverse("hello")
-        
-        expect(result).to eql("olleh")
-    end
+        it "returns olleh for hello" do
+            string_mock = double("string_mock")
+            expect(string_mock).to receive(:reverse).and_return("olleh")
+            
+            allow(Solver).to receive(:new).and_return(string_mock)
+            
+            solver = Solver.new
+            result = solver.reverse("hello")
+            
+            expect(result).to eql("olleh")
+        end
     end
 
     describe "#factorial" do # test for factorial method
-    it "returns 120 for 5" do
-        mockInteger = double("mockInteger")
-        expect(mockInteger).to receive(:factorial).and_return(120)
+        it "returns 120 for 5" do
+            mockInteger = double("mockInteger")
+            expect(mockInteger).to receive(:factorial).and_return(120)
 
-        allow(Solver).to receive(:new).and_return(mockInteger)
-        solver = Solver.new
-        expect(solver.factorial(5)).to eql(120)
-    end
+            allow(Solver).to receive(:new).and_return(mockInteger)
+            solver = Solver.new
+            expect(solver.factorial(5)).to eql(120)
+        end
```

Comment:
Good job! with the passing tests 🤓💯

However, according to the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/testing/tdd_project.md), please add the `#factorial` method test that handles:

- A negative value that raises an exception as an expectation ☺👍

![image](https://github.com/smunoz1988/tdd-test-project/assets/78251927/e1af7a1d-5921-4dad-a0be-cc85b182b6ae)

Reply:
@abdulhamiid  and @danielamoreno699  Changes done!